docs/src/further_topics/netcdf_io.rst

```diff
@@ -235,3 +235,70 @@ Worked example:
     >>> my_coord.ignore_axis = True
     >>> print(guess_coord_axis(my_coord))
     None
+
+Multiple Coordinate Systems and Ordered Axes
+--------------------------------------------
+
+Traditionally, the coordinate system for coordinate of a data variable
+is specified by variable a single name in the ``grid_mapping`` attribute,
+e.g.:
+
+::
+
+    grid_mapping = 'latitude_longitude`
+
+Since version `1.8 of the CF Conventions
+<https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#grid-mappings-and-projections>`_
+, there has been support for the concept of **ordered axes** in the definition
+of a coordinate system which allows for more explicit specification of the
+associated coordinate variables.
+
+This is achieved via use of an extended syntax in the ``grid_mapping``
+attribute of a data variable:
+
+::
+
+    <gridMappingVariable>: <coordinatesVariable> [<coordinatesVariable> …] [<gridMappingVariable>: <coordinatesVariable>…]
```

Comment:
I get that this is fully generic, but that makes it hard to take in at a glance. It needs an example. Either in addition or instead.